tests/observability/metrics/test_vms_metrics.py

```diff
@@ -471,3 +471,13 @@ def test_kubevirt_vmi_non_evictable(
             metric_name="kubevirt_vmi_non_evictable",
             expected_value="1",
         )
+
+
+class TestSSPVmRbdBlockVolumeWithoutRxBounce:
+    @pytest.mark.polarion("CNV-11760")
+    def test_kubevirt_ssp_vm_rbd_block_volume_without_rxbounce(self, prometheus, ocs_rbd_vm):
```

Comment:
It looks like it is already running as part of the [T1 suite ](https://github.com/kubevirt/ssp-operator/blob/c48538539eaeba9c38d26abac582a44169462128/tests/monitoring_test.go#L284) , and is passing as expected as well.

Do you still think we need this ?  (you might want to add a test_id for the test that is missing)